src/forms/inputs/checkbox-group.js

```diff
@@ -34,6 +36,7 @@ import {
  * @param {Object} meta - A `redux-form` [meta](http://redux-form.com/6.5.0/docs/api/Field.md/#meta-props) object
  * @param {Array} options - An array of checkbox values (strings, numbers, or key-value pairs)
  * @param {Object} [checkboxInputProps={}] - An object of key-value pairs representing props to pass down to all checkbox inputs
+ * @param {Boolean} [useDropdown] - A boolean that determines whether the checkbox options are displayed in a dropdown container or not (optional, default `false`)
```

Comment:
I'm pretty hesitant on the naming here. `useX` in React-land now has a very specific set of expectations with it after the release of hooks. I'd prefer to leave that prefix for custom hooks and not for prop names, if possible.